ifrs17-template/Report/InteractiveReports/Accruals.ipynb

```diff
@@ -0,0 +1,92 @@
+{
+    "metadata": {
```

Comment:
Also here... remove all those notebooks with reports and implement a report selector.

Reply:
@dtrzesniak we have tried to do so but failed. @andrey-katz-systemorph do you remember what was the problem? I suspect the problem was in using generic types.

Reply:
After yesterday investigation I assume we can resolve this conversation